diff -up gnome-disk-utility-3.14.0/data/gnome-disk-image-mounter.desktop.in.encoding gnome-disk-utility-3.14.0/data/gnome-disk-image-mounter.desktop.in
--- gnome-disk-utility-3.14.0/data/gnome-disk-image-mounter.desktop.in.encoding	2015-05-22 17:07:05.656535428 -0400
+++ gnome-disk-utility-3.14.0/data/gnome-disk-image-mounter.desktop.in	2015-05-22 17:07:15.195647640 -0400
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 _Name=Disk Image Mounter
 _Comment=Mount Disk Images
 Exec=gnome-disk-image-mounter %U
diff -up gnome-disk-utility-3.14.0/data/gnome-disk-image-writer.desktop.in.encoding gnome-disk-utility-3.14.0/data/gnome-disk-image-writer.desktop.in
--- gnome-disk-utility-3.14.0/data/gnome-disk-image-writer.desktop.in.encoding	2015-05-22 17:07:27.566793165 -0400
+++ gnome-disk-utility-3.14.0/data/gnome-disk-image-writer.desktop.in	2015-05-22 17:07:33.078858005 -0400
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 _Name=Disk Image Writer
 _Comment=Write Disk Images to Devices
 Exec=gnome-disks --restore-disk-image %U
